src/main/java/sirius/biz/storage/layer2/mongo/ProcessChangedEntitiesLoop.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.storage.layer2.mongo;
+
+import sirius.biz.storage.util.StorageUtils;
+import sirius.db.mongo.Mango;
+import sirius.db.mongo.Mongo;
+import sirius.kernel.async.BackgroundLoop;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.PriorityParts;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.health.Exceptions;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Processes {@link MongoBlob blobs} which have been marked as changed.
+ * <p>
+ * The actual processing is performed by the predefined {@link MongoBlobChangeHandler handlers}
+ */
+@Register(framework = MongoBlobStorage.FRAMEWORK_MONGO_BLOB_STORAGE)
+public class ProcessChangedEntitiesLoop extends BackgroundLoop {
```

Comment:
Maybe include the word Blob in the Class Name and the Loop getName?

Reply:
Makes perfectly sense.
@andyHa : OK to change the delete loop to match this naming?